extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -547,7 +547,7 @@ public void statusChanged(String taskId, TaskStatus status)
             {
               notices.add(new RunNotice());
             }
-          }, MoreExecutors.sameThreadExecutor()
+          }, Execs.sameThreadExecutor()
```

Comment:
`Execs.sameThreadExecutor()` should be on the next line.

Reply:
fixed